nrf_modem/doc/gnss_interface.rst

```diff
@@ -10,6 +10,10 @@ GNSS interface
 `Global Navigation Satellite System (GNSS)`_ interface in the Modem library is used to control the GNSS module.
 The interface configures and fetches data from the GNSS module and writes `A-GPS`_ data to the GNSS module.
 
+.. note::
+
+   Before GNSS module can be configured or started, GNSS must be enabled in modem system mode and activated in modem functional mode.
```

Comment:
How about in more active mode? Something like this:
You need to enable GNSS in modem system mode and activate it in modem functional mode before starting or configuring the GNSS module.